draft-ietf-moq-requirements.md

```diff
@@ -360,7 +359,7 @@ To enable use cases where receivers may wish to address a particular time of med
 
 ### Authentication & Authorisation
 
-Whilst QUIC and conversely TLS supports the ability for mutual authentication through client and server presenting certificates and performing validation, this is infeasible in many use cases where provisioning of client TLS certificates is unsupported or infeasible. Thus, support for a primitive method of authentication between MoQ entities SHOULD be included to authenticate entities between one another, noting that implementations and deployments should determine which authorisation model if any is applicable.
+Whilst QUIC and conversely TLS supports the ability for mutual authentication through client and server presenting certificates and performing validation, this is infeasible in many use cases where provisioning of client TLS certificates is unsupported or impractible. Thus, support for a primitive method of authentication between MoQ entities SHOULD be included to authenticate entities between one another, noting that implementations and deployments should determine which authorisation model if any is applicable.
```

Comment:
Do you mean "impractical"?

Reply:
@fiestajetsam - indeed I do. I'll fix this, and verify that it's fixed, and then merge the PR. Thanks for your review!